### ISSUE TITLE
unclear purpose of `IPolicy.ClaimState.Declined`

### ISSUE THREAD

**Issue by rapidddenis** [etherisc/gif-contracts#103]
In https://github.com/etherisc/gif-contracts/blob/b58fd27a34452e062d10b175f3bdd7ee16b9d1a7/contracts/modules/PolicyController.sol

`ClaimState.Declined` have no usage outside of `PolicyController` where it influences nothing.

**Comment by matthiaszimmermann** [comment]
the semantics of declining a claim is a (non-parametric) product where the policy holder would be allowed to create a claim. in such a situation the insurer needs a way to flag invalid claims. 

this is done via the `ClaimState.Declined` state and can be set via base contract `Product._declineClaim` in the gif-interface repository.

**Comment by rapidddenis** [comment]
APPLIED-> CONFIRMED->CLOSED / APPLIED->DECLINED. No close is needed for declined claim? Two "end" states? If claim is closed after declining then anyone who is trying to see the whole picture will have to dig into the logs?

**Comment by matthiaszimmermann** [comment]
yes there are two end states for claims. for applications there are even 3 end states.

to see the whole picture for a specific policy the `InstanceSerive` functions `claims` and `getClaim` may be used. corresponding functions are provided for payouts.

**Comment by rapidddenis** [comment]
Current implementation requires declined claim to be closed, otherwise policy will not close. So declined is not the end state.
